README.md

```diff
@@ -1,14 +1,13 @@
 # Ballerina Shopify connector
 
-[![Build](https://github.com/ballerina-platform/module-ballerinax-shopify/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify/actions/workflows/ci.yml)
-[![Trivy](https://github.com/ballerina-platform/module-ballerinax-shopify/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify/actions/workflows/trivy-scan.yml)
-[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-shopify/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify/actions/workflows/build-with-bal-test-graalvm.yml)
-[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-shopify.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify/commits/master)
-[![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/shopify.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%shopify)
+[![Build](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/actions/workflows/ci.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/actions/workflows/trivy-scan.yml)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/actions/workflows/build-with-bal-test-graalvm.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-shopify.admin.svg)](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/commits/master)
+[![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/shopify.admin.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%shopify.admin)
 
 ## Overview
 
-
 [Shopify](https://www.shopify.com) is a leading e-commerce platform that enables users to create, manage, and grow their online stores, offering secure transactions, inventory management, and seamless integrations.
 
 The Ballerina Shopify connector allows developers to interact with the [Shopify Admin REST APIs](https://shopify.dev/docs/api/admin-rest). It provides an easy-to-use interface for managing products, orders, customers, and other essential e-commerce functionalities. The module supports Shopify API versions up to the `2025-01` release.
```

Comment:
```suggestion
The Ballerina Shopify admin connector allows developers to interact with the [Shopify Admin REST APIs](https://shopify.dev/docs/api/admin-rest). It provides an easy-to-use interface for managing products, orders, customers, and other essential e-commerce functionalities. The module supports Shopify API versions up to the `2025-01` release.
```

Reply:
Check other similar places

Reply:
Fixed in [58d5d58](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/pull/8/commits/58d5d58a4eb004e23afdbef6bdb24c38be8d3bcb)